client/my-sites/importer/section-import.jsx

```diff
@@ -95,7 +95,17 @@ class SectionImport extends Component {
 	handleStateChanges = () => {
 		this.props.siteImports.map( ( importItem ) => {
 			const { importerState, type: importerId } = importItem;
-			this.trackImporterStateChange( importerState, importerId );
+			let eventProps = {};
+
+			// Log more info about upload failures
+			if ( importerState === appStates.UPLOAD_FAILURE ) {
+				eventProps = {
+					error_code: importItem.errorData.code,
+					error_type: importItem.errorData.type,
+				};
+			}
+
```

Comment:
You could add more specific event props for `IMPORT_FAILURE` state as well to get more visibility into different failures, although I saw there's a more detailed backend tracking for that already.